src/core/store/modules/tokens/tokens.selectors.ts

```diff
@@ -86,26 +86,32 @@ const selectWalletTokensStatus = createSelector(
 
 /* --------------------------------- Helper --------------------------------- */
 interface CreateTokenProps {
-  tokenData: Token;
+  tokenData?: Token;
```

Comment:
We were expecting this to be `undefined` below but not handling it when it was, tried to just return `null` in the `createToken()` but it's a bit entangled and was breaking unrelated parts of the code. Potentially worth a refactor in a different PR